packages/jw-player/src/mixins/ChapterMixin.js

```diff
@@ -0,0 +1,127 @@
+import { css, html, render } from 'lit';
+
+// eslint-disable-next-line func-names
+const chapterNavigationTemplate = function (chapters) {
+  return html`
+    <a class="close" @click=${this.__toggleChapterNavigation.bind(this)}>&#10005;</a>
+    <ul class="chapter-navigation-list">
+      ${chapters.map(
+        (chapter, index) =>
+          html`
+            <li data-index=${index} @click=${this.__chapterSeek.bind(this)}>
+              <a>${chapter.data.text}</a>
+            </li>
+          `
+      )}
+    </ul>
+  `;
+};
+
+export function ChapterMixin(BaseClass) {
+  class Mixin extends BaseClass {
+    __path = 'https://cxl.com/institute/wp-content/plugins/cxl-jwplayer/';
```

Comment:
Hardcoded path, should be handled via PHP, using either:

`wp_add_inline_script` or `wp_localize_script`

Reply:
I added the `plugin-path` attribute to specify this directory.